docker-compose.yml

```diff
@@ -55,10 +60,16 @@ services:
       - mysql
       - elasticsearch
       - redis
-      # Drop kumascript link
     ports:
       - "8001:8000"
 
+  # ssr is a Node server that performs server-side rendering of our React UI
+  ssr:
+    <<: *worker
+    command: node kuma/javascript/ssr-server.js
+    ports:
+      - "8002:8000"
```

Comment:
If we go with my suggestion above, we'd have to explicitly state that `ssr` has no dependencies, like this:
```
depends_on: []
```

Reply:
Done. Thanks. (I never new how to create an empty list in yaml...)